apps/avifenc.c

```diff
@@ -36,18 +36,19 @@ typedef struct
     avifBool qualityAlphaIsConstrained; // true if qualityAlpha explicitly set by the user
     int overrideQuality;
     int overrideQualityAlpha;
-    avifBool autoProgressive;
-    int speed;
+    avifBool autoProgressive; // automatic layered encoding (progressive) with single input
```

Comment:
Should this be renamed back to `progressive` to match the flag "--progressive"? `autoProgressive` is fine but maybe it was a leftover.